### PR TITLE
Add Devcontainers Config and Rewrite PhlexUI Documentation

### PR DESCRIPTION
**Pull request by cirdes**
### This PR includes the following updates:

1. **Added Devcontainers Configuration**: 
   - Configured Devcontainers to streamline the development environment setup and improve the developer experience.

2. **Rewritten Documentation for PhlexUI Components**: 
   - Updated the documentation for all PhlexUI components to reflect the new syntax.

Before we can merge this PR, we need to merge this fix https://github.com/PhlexUI/phlex_ui/pull/61, release a new version of PhlexUI, and update the Gemfile in this PR.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -86,5 +86,6 @@ gem "rack-www" # used to redirect www to non-www subdomain
 gem "rollbar"
 
 gem "phlex-rails"
-gem "phlex_ui"
+# gem "phlex_ui"
 # gem "phlex_ui", path: "../phlex_ui"
+gem "phlex_ui", git: "https://github.com/cirdes/phlex_ui.git", branch: "fix-clipboard"
```
Comment (iseth) [inline Gemfile:91]: This should be using the main branch

Comment (cirdes) [reply Gemfile]: @iseth , done!

File: app/views/docs/badge_view.rb [hunk 2]
```diff
@@ -78,7 +78,7 @@ def view_template
 
   def components
     [
-      Docs::ComponentStruct.new(name: "PhlexUI::Badge", source: "https://github.com/PhlexUI/phlex_ui/blob/main/lib/phlex_ui/badge.rb", built_using: :phlex)
```
Comment (iseth) [inline app/views/docs/badge_view.rb:81]: If we are using `Badge` maybe we should specify that `include Phlex` need to be in `application_view.rb` or use `Phlex::Badge`

Comment (cirdes) [reply app/views/docs/badge_view.rb:81]: @iseth , sure! I will try to find the remaining "PhlexUI::" references and add to the installation guide the instruction to `include Phlex` into `application_view.rb`

Anything else?

Comment (cirdes) [reply app/views/docs/badge_view.rb:81]: Screenshot 2024-07-06 at 9 54 31 AM

@iseth , I added the instruction to add PhlexUI module.

**Review by iseth — APPROVED** [review]
looks great!!

**Pull request merged**